ansible/roles/xl_auth/tasks/main.yml

```diff
@@ -81,19 +104,25 @@
 - name: run 'flask import_data'
   docker_container:
     name: xl_auth_import_data
+    hostname: xl_auth
     image: "{{ xl_auth_docker }}"
+    recreate: yes
     interactive: yes
     tty: yes
     detach: no
     command: "import_data --verbose --admin-email libris@kb.se --wipe-permissions
-             --send-password-resets"
+             {{ '--send-password-resets' if inventory_hostname == 'login.libris.kb.se' else '' }}"
+    log_driver: syslog
```

Comment:
Why do we use syslog here?

Reply:
It's so that data import output will be stored "somewhere retrievable". And a bit more nicely formatted than the JSON blob you get when running Ansible with `-v`!

Reply:
Check.